tiledb/sys/src/group.rs

```diff
@@ -0,0 +1,207 @@
+use crate::capi_enum::{tiledb_object_t, tiledb_query_type_t};
+use crate::tiledb_datatype_t;
+use crate::types::{
+    capi_return_t, tiledb_config_t, tiledb_ctx_t, tiledb_group_t,
+    tiledb_string_t,
+};
+
+extern "C" {
+    #[doc = " Creates a new TileDB group.\n\n **Example:**\n\n @code{.c}\n tiledb_group_create(ctx, \"my_group\");\n @endcode\n\n @param ctx The TileDB context.\n @param group_uri The group URI.\n @return `TILEDB_OK` for success and `TILEDB_ERR` for error."]
```

Comment:
Please remove the all the doc strings and then make sure the first two functions are `tiledb_group_alloc` followed by `tiledb_group_free`.